randovania/games/prime1/gui/dialog/prime_cosmetic_patches_dialog.py

```diff
@@ -15,18 +37,44 @@ def __init__(self, parent: QWidget, current: PrimeCosmeticPatches):
         self.setupUi(self)
         self._cosmetic_patches = current
 
+        # Build dynamically preview color squares for suits
+        suit_layouts = [ 
+            self.power_suit_color_layout, self.varia_suit_color_layout, 
+            self.gravity_suit_color_layout, self.phazon_suit_color_layout
+        ]
+        self.suit_color_preview_squares = []
+        for i, suit_colors in enumerate(SUIT_DEFAULT_COLORS):
```

Comment:
```python
for suit_layout, suit_colors in zip(suit_layouts, SUIT_DEFAULT_COLORS):
```

Reply:
Didn't know about `zip`, that's super cool